src/main/io/lights.c

```diff
@@ -0,0 +1,84 @@
+#include "stdbool.h"
+#include "stdint.h"
+#include "stdlib.h"
+
+#include "platform.h"
+
+#include "common/utils.h"
+
+#include "drivers/time.h"
+#include "drivers/io.h"
+
+#include "rx/rx.h"
+
+#include "fc/config.h"
+#include "fc/rc_controls.h"
+#include "fc/rc_modes.h"
+#include "fc/runtime_config.h"
+
+#include "config/feature.h"
+
+#include "io/lights.h"
+
+#ifdef LIGHTS
+
+static IO_t lightsIO = DEFIO_IO(NONE);
+static bool lights_on = false;
+
+#ifdef FAILSAFE_LIGHTS
+  static timeUs_t last_status_change = 0;
+#endif
+
+bool lightsSetStatus(bool status)
+{
+  if (status != lights_on) {
+    lights_on = status;
+    IOWrite(lightsIO, lights_on);
+    return(true);
+  } else
+    return(false);
+}
```

Comment:
Please use standard 4 space indentation.

Reply:
Done. I need to change my auto-indentation settings for iNav it is not the same I use for other projects.